docs/changelogs/CHANGELOG-2.24.md

```diff
@@ -0,0 +1,157 @@
+- Fix an issue in the `kubermatic-installer mirror-images` command, which led to failure on the mla-consul chart ([#12513](https://github.com/kubermatic/kubermatic/pull/12513))
+- Fix an issue where IPv6 IPs were being ignored when determining the address of a user cluster ([#12505](https://github.com/kubermatic/kubermatic/pull/12505))
+- Fix node-labeller controller not applying the `x-kubernetes.io/distribution` label to RHEL nodes ([#12751](https://github.com/kubermatic/kubermatic/pull/12751))
+- Fix reconcile loop for `seed-proxy-token` Secret on Kubernetes 1.27 ([#12557](https://github.com/kubermatic/kubermatic/pull/12557))
+- Increase memory limit of kube-state-metrics addon to 600Mi ([#12692](https://github.com/kubermatic/kubermatic/pull/12692))
+- KKP's CA bundle was not used when performing Application-related operations like installing a Helm chart from a private OCI registry ([#12514](https://github.com/kubermatic/kubermatic/pull/12514))
+- `kubermatic-installer` will now validate the existing MinIO filesystem before attempting a `kubermatic-seed` stack installation ([#12477](https://github.com/kubermatic/kubermatic/pull/12477))
+
+### Updates
+
+- Update Vertical Pod Autoscaler to 0.14.0 ([#12604](https://github.com/kubermatic/kubermatic/pull/12604))
+- Update `d3fk/s3cmd` to version (latest "arch-stable") with `fb4c4dcf` hash ([#12640](https://github.com/kubermatic/kubermatic/pull/12640))
+- Update cert-manager to 1.12.2 ([#12443](https://github.com/kubermatic/kubermatic/pull/12443))
+- Update curl in `kubermatic/util` image and `mla/grafana` chart to 8.4.0 (CVE-2023-38545 and CVE-2023-38546 do not affect KKP) ([#12694](https://github.com/kubermatic/kubermatic/pull/12694))
+- Update to `quay.io/kubermatic/util:2.3.1` as helper image (includes curl version patched against CVE-2023-38545 and CVE-2023-38546) ([#12726](https://github.com/kubermatic/kubermatic/pull/12726))
+- Update etcd for user clusters to 3.5.9 ([#12453](https://github.com/kubermatic/kubermatic/pull/12453))
+- Update KubeVirt chart for the installer local command to the latest release v1.0.0 ([#12470](https://github.com/kubermatic/kubermatic/pull/12470))
+- Update metering Prometheus to next LTS version 2.45.0 ([#12532](https://github.com/kubermatic/kubermatic/pull/12532))
+- Update metrics-server for all deployments to v0.6.4 ([#12516](https://github.com/kubermatic/kubermatic/pull/12516))
+- Update nginx-ingress-controller to v1.9.3 (fixes CVE-2023-44487, HTTP/2 rapid reset attack) ([#12712](https://github.com/kubermatic/kubermatic/pull/12712))
+- Update supported Kubernetes releases for EKS/AKS ([#12579](https://github.com/kubermatic/kubermatic/pull/12579))
+- Update telemetry-agent to v0.4.1 ([#12572](https://github.com/kubermatic/kubermatic/pull/12572))
+- Update to controller-runtime 0.16.1 / Kubernetes 1.28 ([#12609](https://github.com/kubermatic/kubermatic/pull/12609))
+- Update to Go 1.21.3 ([#12697](https://github.com/kubermatic/kubermatic/pull/12697))
```

Comment:
`Update to <library>` => `Update <library> to`, then re-sort all the update bulletpoints.